### PR TITLE
Fix edge case of set scheduled SCRs at bootstrap 

### PR DESCRIPTION
**Pull request by SebastianMarian**
* Fixed some misleading logs
* Fixed an edge case situation which wrongly set scheduled SCRs for the current block when a node starts in epoch

### CODE REVIEW

File: process/block/preprocess/scheduledTxsExecution.go [hunk 1]
```diff
@@ -488,6 +488,15 @@ func (ste *scheduledTxsExecution) IsScheduledTx(txHash []byte) bool {
 	return ok
 }
 
+func getNumScheduledSCRs(mapScheduledSCRs map[block.Type][]data.TransactionHandler) int {
```
Comment (AdoAdoAdo) [inline process/block/preprocess/scheduledTxsExecution.go:491]: :+1:

**Review by AdoAdoAdo — DISMISSED** [review]

File: epochStart/bootstrap/startInEpochScheduled.go [hunk 2]
```diff
@@ -355,8 +364,12 @@ func (ses *startInEpochWithScheduledDataSyncer) getScheduledTransactionHashes(he
 
 	scheduledTxs := make(map[string]struct{})
 	for _, mb := range miniBlocks {
+		if mb.GetReceiverShardID() != header.GetShardID() {
+			continue
+		}
 		for _, txHash := range mb.TxHashes {
 			scheduledTxs[string(txHash)] = struct{}{}
+			log.Debug("startInEpochWithScheduledDataSyncer.getScheduledTransactionHashesWithDestMe", "hash", txHash)
```
Comment (sasurobert) [inline epochStart/bootstrap/startInEpochScheduled.go:372]: this is a little too much log.debug - please delete

Comment (SebastianMarian) [reply epochStart/bootstrap/startInEpochScheduled.go:372]: There is only for scheduled txs, if they exist, from one single block at bootstrap -> start in epoch. So, it is printed only once, but it helps for debugging if needed!

Comment (sasurobert) [reply epochStart/bootstrap/startInEpochScheduled.go:372]: ah. I missed that it is only start in epoch. alright.

**Review by sasurobert — DISMISSED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

File: process/block/postprocess/feeHandler.go [hunk 3]
```diff
@@ -61,7 +61,7 @@ func (f *feeHandler) GetAccumulatedFees() *big.Int {
 // GetDeveloperFees returns the total accumulated developer fees
 func (f *feeHandler) GetDeveloperFees() *big.Int {
 	f.mut.RLock()
-	developerFees := f.developerFees
+	developerFees := big.NewInt(0).Set(f.developerFees)
```
Comment (iulianpascalau) [inline process/block/postprocess/feeHandler.go:64]: :+1:

**Review by iulianpascalau — DISMISSED** [review]

File: process/block/postprocess/feeHandler.go (hunk 3 above)
Comment (sasurobert) [reply process/block/postprocess/feeHandler.go:64]: +1

**Review by sasurobert — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed .

**Review by gabi-vuls — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**